src/task2/userHandler.js

```diff
@@ -1,8 +1,37 @@
 const express = require('express');
 const users = require('./router/users');
+const logger = require('./logging/logger');
+const { CustomError } = require('./data/CustomError');
+const { getRequestLogMessage, getErrorLogMessage } = require('./logging/loggingMessagesProvider');
 const app = express();
 app.use(express.json());
 
+process
+    .on('unhandledRejection', (reason) => {
+        logger.error('Unhandled promise exception occurred with reason:\n' + reason.stack)
+        process.exit(1);
+    })
+    .on('uncaughtException', err => {
+        logger.error('Uncaught exception occurred. ' + err.stack);
+        process.exit(1);
+    });
+
+app.use('/users', function (req, res, next) {
```

Comment:
middleware should log any request against service, so first param '/users' can be removed